rewriter/tests/mmap_loop/main.c

```diff
@@ -0,0 +1,53 @@
+/*
+RUN: sh -c 'if [ ! -s "mmap_loop_call_gates_0.ld" ]; then echo "No link args as expected"; exit 0; fi; echo "Unexpected link args"; exit 1;'
+RUN: %binary_dir/tests/mmap_loop/mmap_loop_main_wrapped | diff %S/Output/mmap_loop.out -
+*/
+#include "mmap_loop.h"
+#include <assert.h>
+#include <ia2.h>
+#include <math.h>
+#include <stdio.h>
+
+#define IA2_DEFINE_TEST_HANDLER
+#include "test_fault_handler.h"
+
+/*
+    This program tests that mmap and heap allocations are handled properly.
+*/
+
+INIT_RUNTIME(2);
+#define IA2_COMPARTMENT 1
+#include <ia2_compartment_init.inc>
+
+int main() {
+  // wait to be traced
+  sleep(2);
```

Comment:
This is only necessary for now because our memory-map tracking demo ptraces an existing process instead of doing fork+exec. The actual runtime will fork so the tracee is its child.